sync/doc/index.rst

```diff
@@ -347,6 +347,14 @@ Asynchronous work
   * ``identity_key`` key uniquely identifying the job, if specified and a job
     with the same key has not yet been run, the new job will not be added.
 
+
+Attachments
+~~~~~~~~~~~
+
+Used to send attachments to an external system, which receive the picture/document as a URL
+
+* ``attachment.get_public_url()``: attachment is odoo attachment object
```

Comment:
```suggestion
* ``attachment.get_public_url()``: generates access url. Can be used to pass attachments to an external system as url, instead of direct uploading the content. 
```